package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "who-wants-to-be-a-javascriptaire",
+  "version": "1.0.0",
+  "description": "<div align=\"center\">",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prettier": "prettier \"**/*.+(js)\"",
+    "check-format": "npm run prettier -- --list-different",
+    "format": "npm run prettier -- --write"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run check-format"
+    }}
+  
```

Comment:
on running `npm run format` this will throw error  `sh: prettier: command not found`

Because prettier is not installed. Please install it as dev dependency

It worked for you probably because you have prettier installed globally on your machine